src/main/java/org/jabref/logic/importer/fileformat/CffImporter.java

```diff
@@ -44,19 +44,29 @@ public String getId() {
 
     @Override
     public String getDescription() {
-        return "Importer for the CFF format. Is only used to cite software, one entry per file.";
+        return "Importer for the CFF format. Is only used to cite software, one entry per file. Can also " +
```

Comment:
I think, `cite a preferred citation` is misleading. Maybe just shorten to

    Importer for the CFF format, which is intended to make software and datasets citable.